.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
-.env.*
\ No newline at end of file
+.env.*
+package-lock.json
```

Comment:
You dont actually want to ignore the package.json, or whoever clone your repo will have to manually find and install everything.